### PR TITLE
#166790057 Implements liking and disliking of Articles

### PR DESCRIPTION
**Pull request by mnzube**
#### What does this PR do?
This PR allows a user to like and/or dislike articles.
#### Description of Task to be completed?
- add actions and reducers
- add icons to like or dislike
- added code for the component
- introduce tests
All this will allow users to like or dislike articles,
#### How should this be manually tested?
After cloning the repository run  `npm i` to install dependencies then `npm run dev`
this will redirect you to the profile page. Select an article and at the bottom of an article,
you will find a thumbs up and down icons which depict liking or disliking an article.
Click on either to like or dislike.
#### What are the relevant pivotal tracker stories?
[#166790057](https://www.pivotaltracker.com/story/show/166790057)
#### Screenshots (if appropriate)
likedislike

### CODE REVIEW

File: src/redux/store/initialStates/userInitialState.js [hunk 1]
```diff
@@ -52,6 +52,20 @@ module.exports = {
     errors: {},
     data: {},
   },
+  likes: {
```
Comment (shaluchandwani) [inline src/redux/store/initialStates/userInitialState.js:59]: Always add the initial state at the end, this won't create conflicts later.

**Review by Joe-Joseph — COMMENTED** [review]
Good work on your PR @mnzube, kindly adjust the like and dislike buttons to come after the article body has displayed.

File: src/redux/store/initialStates/userInitialState.js (hunk 1 above)
Comment (mnzube) [reply src/redux/store/initialStates/userInitialState.js:59]: Alright Shalu , changed this.

**Review by shaluchandwani — CHANGES_REQUESTED** [review]
Good work on the PR @mnzube, although I found out some issues please address them.

1. Change you commit message to finishes.
2. When I am not logged in, I cannot click on the likes and dislikes button. It should actually throw me an error that " Please login first to perform the action".

**Review by kagabof — APPROVED** [review]

**Review by Joe-Joseph — APPROVED** [review]
Good work on this @mnzube

**Pull request merged**